README.md

```diff
@@ -36,3 +36,35 @@ To run the project locally, you can use the `docker-compose.yml` file as such:
 ```
 $ docker-compose up -d
 ```
+
+## Slack App setup
+
+You should first create [a new app](https://api.slack.com/authentication/basics).
+
+Give this app the following scopes:
+
+- `chat:write`
+- `channels:read`
+- `commands`
+
+Then create commands for this app. The app supports the following commands, you
+don't have to register them all:
+
+- `/poll`
+- `/poll-open`
+- `/poll-unique`
+- `/poll-anon`
+- `/poll-anon-unique`
+- `/poll-build`
+- `/report`
+- `/cri-photo`
```

Comment:
List of commands I've found by reading throught the code, I might have missed some. Tell me if anything's missing!